### PR TITLE
Feature: modals for content and share

### PR DESCRIPTION
**Pull request by edbrett**
Add two modals using modals helper view:
- set modal for interventions on cost packages view
- set basic modal for share modal (designs pending)
- Create helper for data action targets to trigger modal events

### CODE REVIEW

File: app/assets/javascripts/helpers/action_click.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+(function(App) {
```
Comment (geriux) [inline app/assets/javascripts/helpers/action_click.js:1]: I don't think a helper for click actions would be useful for what we'll be using it for, what do you think if we just add a listener in the Share Modal, to bind a global click event to a specific class in this case for the footer link?

File: app/assets/javascripts/templates/modal_share.hbs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <div class="share-social">
+          <ul class="c-btn-container -j-between -a-center">
+            <li>
+              <a href="https://twitter.com/share?url={{this.urlEncoded}}" target="_blank" class="btn -light -with-icon -twitter js-btn-popup">
+                <svg class="c-icon -medium"><use xlink:href="#icon-twitter"></use></svg>
+                <span>Twitter</span>
+              </a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/shareArticle?mini=true&url={{this.urlEncoded}}&title=Grow%20Asia%20Counter&summary=The Grow Asia Counter allows users to estimate their greenhouse gas emissions of different agricultural management scenarios.&source=" target="_blank" class="btn -light -with-icon -linkedin js-btn-popup">
```
Comment (geriux) [inline app/assets/javascripts/templates/modal_share.hbs:27]: We gotta update this param values :)

File: app/assets/javascripts/views/share_modal.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                   ',left='   + left;
+
+      window.open(url, 'Share this', opts);
+    },
+
+    onClickCopy: function(e) {
+      e && e.preventDefault();
+      this.$copyInput.select();
+
+      try {
```
Comment (geriux) [inline app/assets/javascripts/views/share_modal.js:77]: Niceeee!!

File: app/views/dashboards/show.html.erb [hunk 4]
```diff
@@ -97,6 +97,11 @@
     <div class="column small-12 medium-6">
       <p class="text">Scaling up this priority package would cost less than scaling up a package of all interventions needed reach the WHA targets. However, the targets would not be achieved under this low-cost scale-up scenario.</p>
     </div>
+    <div class="columns small-12">
+      <button class="text -button js-btn-action" data-action="Info:toggle">
```
Comment (geriux) [inline app/views/dashboards/show.html.erb:101]: Here, like I mentioned above, if we remove the click action helper, we'll avoid to have global events names in the HTML. So adding a global class like js-modal-action or whatever and then binding that in the helper would work too. What  do you think?

**Review by geriux — CHANGES_REQUESTED** [review]
I've added some comments, let me know what you think!

**Comment by geriux** [comment]
And for specific modals like the interventions one, I'd just bind the event in the view to open the modal.

File: app/assets/javascripts/views/share_modal.js (hunk 3 above)
Comment (mcorriga-lenovo) [reply app/assets/javascripts/views/share_modal.js]: **Verdict:** fixed.

**Commit:** 1e494e7098 (`fix(qira-core): omit the cloud Qira bearer on loopback MCP`), kept under the file-size ratchet in adaec98e37.

`connect_inner` now passes `direct_url::qira_bearer_for_url`, which yields an empty token for loopback destinations. `McpClient::try_rpc_send` omits `Authorization` when that token is empty; `X-Api-Token` is still sent.

**Test:** `loopback_direct_url_omits_cloud_qira_bearer` fails without the strip (stub records `Authorization` / `X-Api-Token`; asserts no `cloud-qira-secret`, still has `launch-token`).

**Pull request merged**